init.bzl

```diff
@@ -62,3 +65,7 @@ def init(python_interpreter = None, python_interpreter_target = None):
     protobuf_deps()
 
     bazel_skylib_workspace()
+
+    m4_register_toolchains(version = "1.4.18")
+    bison_register_toolchains(version = "3.3.2")
+    flex_register_toolchains(version = "2.6.4")
```

Comment:
Can you move these to dependency_support.bzl

Reply:
I would love to but unfortunately I'd need to load these rules first, which happens in `dependency_support.bzl` 😞. I can do so if you can suggest where I should load the rules in the first place